appinventor/buildserver/src/com/google/appinventor/buildserver/Main.java

```diff
@@ -28,23 +28,23 @@
     boolean isForCompanion = false;
 
     @Option(name = "--inputZipFile", required = true,
-            usage = "the ZIP file of the project to build")
+        usage = "the ZIP file of the project to build")
```

Comment:
I don't understand it... Whenever I change it, the original diff changes as well... Anyway, this is the correct one which contains 4 spaces.